### DISCUSSION TITLE
Questions and Feedback on Using AutomaticCFG with Bolting - Realistic, Lighting Version

### DISCUSSION THREAD

**Discussion by C1c3r0** [Extraltodeus/ComfyUI-AutomaticCFG#37] [General]
Hi there,

First off, I'd like to extend my gratitude for the incredible work on AutomaticCFG. It's truly impressive and has been a game-changer for my projects.

I've been primarily using the Bolting - Realistic, Lighting version from Civitai (https://civitai.com/models/413466/boltning-realistic-lightning-hyper?modelVersionId=488645). The model author suggests using a CFG of 1, but since AutomaticCFG changes how it works, other CFG values also produce interesting results.

I have a few questions and findings I'd like to share:

CFG Adjustment: Should I manually change auto_cfg_ref in the advanced node, or is it adjusted automatically with the checkpoint input?

Preset - "Kickstart" vs "Warp Drive" node: Are these two things equivalent, or should they be used separately?

Temperature Nodes: How do temperature nodes work when set to "0" as suggested by another repository? How does this interact with the "experimental temperature setting" preset?

Clip Set Last Layer Node: Do you recommend using this node in conjunction with temperature settings?

"Tip Top" Node from ComfyUI: Does this node tweak the same parameters as attention modifiers?

Additionally, it would be incredibly helpful to have more detailed explanations on how to use the different nodes. Specifically, it would be great if there was a way to set a preset on the advanced node and easily see what changes each preset makes.

In my experience with the Bolting model, using a CFG of 2 results in some wild but intriguing outputs. This setting seems to produce great quality, but the content often becomes quite literal and exaggerated. For example, when prompting with "chess queen," the generation is very literal. Does this setting turn off negatives? I find the results fascinating but a bit confusing.

Thank you once again for your fantastic work. Any guidance or insights you could provide would be greatly appreciated.

**Comment by Extraltodeus** [comment]
Hello!

Sorry for the late answer I'm only noticing now.

> CFG Adjustment: Should I manually change auto_cfg_ref in the advanced node, or is it adjusted automatically with the checkpoint input?

Unless if, through experimentation, you find an actual interest into changing this setting, I do not recommand to change it as it may expand the range of possible settings in a way that, even me who likes to test a broad range of settings, do not like.

> Clip Set Last Layer Node: Do you recommend using this node in conjunction with temperature settings?

I actually have not tried at all and your question makes me wonder about the possible effects.


> Additionally, it would be incredibly helpful to have more detailed explanations on how to use the different nodes. Specifically, it would be great if there was a way to set a preset on the advanced node and easily see what changes each preset makes.

I know but I'm caught into exploring more and all that I provide is for free so I'm cutting on the tutorials and focus and sharing working solutions.
